workflows/cloudify_system_workflows/plugins.py

```diff
@@ -32,3 +35,43 @@ def _operate_on_plugin(ctx, plugin, action):
         'cloudify_agent.operations.{0}_plugins'.format(action),
         kwargs={'plugins': [plugin]}))
     return graph.execute()
+
+
+@workflow(system_wide=True)
+def update(ctx, update_id, temp_blueprint_id, deployments_to_update, **_):
+    """Execute deployment update for all the given deployments_to_update.
+
+    :param update_id: plugins update ID.
+    :param temp_blueprint_id: temporary blueprint ID that should be used for
+    this workflow only.
+    :param deployments_to_update: deployments to perform the update on, using
+    the temp blueprint ID provided.
+    """
+    err_msg_format = 'Deployment update has failed with execution ID: {}.'
```

Comment:
I prepare it as a function, because you need this var and format it everywhere

Reply:
Made it an inner function, removed the var.